nimbleclient/v1/api/active_directory_memberships.py

```diff
@@ -22,104 +24,131 @@ class ActiveDirectoryMembership(Resource):
     - user                : Name of the Activer Directory user with Administrator's privilege.
     - password            : Password for the Active Directory user.
     - enabled             : Active Directory authentication is currently enabled.
-    """
+    '''
 
     def remove(self, password, user, force=False):
-        """
+        '''
         Leaves the Active Directory domain.
 
         Parameters:
         - id       : ID of the active directory.
         - user     : Name of the Activer Directory user with the privilege to leave the domain.
         - password : Password for the Active Directory user.
         - force    : Use this option when there is an error when leaving the domain.
-        """
+        '''
 
         return self.collection.remove(self.id, password, user, force)
 
     def report_status(self):
-        """
+        '''
         Reports the detail status of the Active Directory domain.
 
         Parameters:
         - id : ID of the active directory.
-        """
+        '''
 
         return self.collection.report_status(self.id)
 
     def test_user(self, name):
-        """
+        '''
         Tests whether the user exist in the Active Directory. If the user is present, then the user's group and role information is reported.
 
         Parameters:
         - id   : ID of the Active Directory.
         - name : Name of the Active Directory user.
-        """
+        '''
 
         return self.collection.test_user(self.id, name)
 
     def test_group(self, name):
-        """
+        '''
         Tests whether the user group exist in the Active Directory.
 
         Parameters:
         - id   : ID of the Active Directory.
         - name : Name of the Active Directory group.
-        """
+        '''
 
         return self.collection.test_group(self.id, name)
 
     def delete(self, **kwargs):
         raise NimOSAPIOperationUnsupported("delete operation not supported")
 
+
 class ActiveDirectoryMembershipList(Collection):
     resource = ActiveDirectoryMembership
     resource_type = "active_directory_memberships"
 
-    def remove(self, id, password, user, force=False):
-        """
+    def remove(self,
+               id,
+               password,
+               user,
+               force=False
+               ):
```

Comment:
This function signature is still fairly small, let's keep it to a single line. Same thing occurs in several places.